package.json

```diff
@@ -226,5 +233,41 @@
       "protractor-cucumber-framework",
       "postcss-cssnext"
     ]
+  },
+  "jest": {
+    "modulePaths": ["<rootDir>/src/components"],
+    "testMatch": [
+      "<rootDir>/src/components/transactions/**/?(*.)+(spec|test).js?(x)",
+      "<rootDir>/test/**/?(*.)+(spec|transactionID.test).js?(x)"
```

Comment:
I would try to make the regexps here as specific as possible to make it easy to understand.  
Something like:
```
"<rootDir>/test/integration/transactionID.test.js?(x)"
```
BTW what is the `?(x)` in the end for?

Reply:
optional to match jsx files